gazebo/rendering/Distortion.cc

```diff
@@ -122,6 +130,11 @@ void Distortion::Load(sdf::ElementPtr _sdf)
   {
     this->dataPtr->compositorName = _sdf->Get<std::string>(compositorName);
   }
+  const std::string legacyMode = "ignition:legacy_mode";
+  if (_sdf->HasElement(legacyMode))
```

Comment:
A debug-level message somewhere, possibly here, might be helpful to inform the user what mode is active. That would inform people who kept the default that this other implementation exists.

Reply:
I added a warning for this :+1: